<ui version="4.0" >
 <class>Main</class>
 <widget class="QMainWindow" name="Main" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>PyNdsGest</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="QLabel" name="labelSize" >
    <property name="geometry" >
     <rect>
      <x>390</x>
      <y>470</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize" >
     <size>
      <width>50</width>
      <height>0</height>
     </size>
    </property>
    <property name="text" >
     <string>Size</string>
    </property>
    <property name="wordWrap" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelPath" >
    <property name="geometry" >
     <rect>
      <x>390</x>
      <y>490</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize" >
     <size>
      <width>50</width>
      <height>0</height>
     </size>
    </property>
    <property name="toolTip" >
     <string/>
    </property>
    <property name="text" >
     <string>Path</string>
    </property>
    <property name="wordWrap" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel3" >
    <property name="geometry" >
     <rect>
      <x>320</x>
      <y>470</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize" >
     <size>
      <width>50</width>
      <height>0</height>
     </size>
    </property>
    <property name="text" >
     <string>Size</string>
    </property>
    <property name="wordWrap" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelTitle" >
    <property name="geometry" >
     <rect>
      <x>390</x>
      <y>410</y>
      <width>340</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize" >
     <size>
      <width>50</width>
      <height>0</height>
     </size>
    </property>
    <property name="text" >
     <string>Title</string>
    </property>
    <property name="wordWrap" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel6" >
    <property name="geometry" >
     <rect>
      <x>320</x>
      <y>490</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize" >
     <size>
      <width>50</width>
      <height>0</height>
     </size>
    </property>
    <property name="text" >
     <string>Path</string>
    </property>
    <property name="wordWrap" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelRelease" >
    <property name="geometry" >
     <rect>
      <x>320</x>
      <y>410</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize" >
     <size>
      <width>50</width>
      <height>0</height>
     </size>
    </property>
    <property name="font" >
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text" >
     <string>Release</string>
    </property>
    <property name="wordWrap" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel5" >
    <property name="geometry" >
     <rect>
      <x>320</x>
      <y>450</y>
      <width>62</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize" >
     <size>
      <width>50</width>
      <height>0</height>
     </size>
    </property>
    <property name="text" >
     <string>Language</string>
    </property>
    <property name="wordWrap" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="pixmapLabel3" >
    <property name="geometry" >
     <rect>
      <x>280</x>
      <y>410</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="pixmap" >
     <pixmap/>
    </property>
    <property name="scaledContents" >
     <bool>true</bool>
    </property>
    <property name="wordWrap" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelLanguage" >
    <property name="geometry" >
     <rect>
      <x>390</x>
      <y>450</y>
      <width>340</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize" >
     <size>
      <width>50</width>
      <height>0</height>
     </size>
    </property>
    <property name="text" >
     <string>Language</string>
    </property>
    <property name="wordWrap" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="liste" >
    <property name="geometry" >
     <rect>
      <x>11</x>
      <y>11</y>
      <width>249</width>
      <height>570</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="pixmapLabel1" >
    <property name="geometry" >
     <rect>
      <x>280</x>
      <y>20</y>
      <width>214</width>
      <height>384</height>
     </rect>
    </property>
    <property name="pixmap" >
     <pixmap>1a.png</pixmap>
    </property>
    <property name="scaledContents" >
     <bool>true</bool>
    </property>
    <property name="wordWrap" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="pixmapLabel2" >
    <property name="geometry" >
     <rect>
      <x>520</x>
      <y>20</y>
      <width>214</width>
      <height>384</height>
     </rect>
    </property>
    <property name="pixmap" >
     <pixmap>1a.png</pixmap>
    </property>
    <property name="scaledContents" >
     <bool>true</bool>
    </property>
    <property name="wordWrap" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxNote" >
    <property name="geometry" >
     <rect>
      <x>460</x>
      <y>470</y>
      <width>55</width>
      <height>29</height>
     </rect>
    </property>
    <property name="maximum" >
     <number>5</number>
    </property>
    <property name="minimum" >
     <number>-1</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="titleEdit" >
    <property name="geometry" >
     <rect>
      <x>320</x>
      <y>510</y>
      <width>160</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboSize" >
    <property name="geometry" >
     <rect>
      <x>320</x>
      <y>540</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text" >
      <string>All</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>8Mo</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>16Mo</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>32Mo</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>64Mo</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>128Mo</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboLangues" >
    <property name="geometry" >
     <rect>
      <x>320</x>
      <y>570</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text" >
      <string>All</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>Français</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>Anglais</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>Chinois</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>Inconnu2</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>Néerlandais</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>Inconnu3</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>Allemand</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>Italien</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>Japonais</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>Inconnu4</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>Inconnu5</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>Inconnu6</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>Espagnol</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>Inconnu7</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>Inconnu8</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>Inconnu9</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>Coréen</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="spinBoxSearchNote" >
    <property name="geometry" >
     <rect>
      <x>480</x>
      <y>540</y>
      <width>55</width>
      <height>29</height>
     </rect>
    </property>
    <property name="maximum" >
     <number>5</number>
    </property>
    <property name="minimum" >
     <number>-1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="updateXML" >
    <property name="geometry" >
     <rect>
      <x>670</x>
      <y>550</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text" >
     <string/>
    </property>
    <property name="icon" >
     <iconset>stock-refresh-bis.png</iconset>
    </property>
    <property name="iconSize" >
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="parcoursRep" >
    <property name="geometry" >
     <rect>
      <x>600</x>
      <y>550</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text" >
     <string/>
    </property>
    <property name="icon" >
     <iconset>hd2-backup-bis.png</iconset>
    </property>
    <property name="iconSize" >
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="envoyer" >
    <property name="geometry" >
     <rect>
      <x>670</x>
      <y>480</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text" >
     <string/>
    </property>
    <property name="icon" >
     <iconset>stock-goto-bottom-bis.png</iconset>
    </property>
    <property name="iconSize" >
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>736</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>Help</string>
    </property>
    <addaction name="separator" />
    <addaction name="helpAboutAction" />
   </widget>
   <widget class="QMenu" name="menuEdit" >
    <property name="title" >
     <string>Edit</string>
    </property>
    <addaction name="separator" />
    <addaction name="separator" />
   </widget>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="fileSaveAction" />
    <addaction name="separator" />
    <addaction name="fileExitAction" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuEdit" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="fileSaveAction" >
   <property name="icon" >
    <iconset>i-floppy-192.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="fileExitAction" >
   <property name="icon" >
    <iconset>gtk-quit.png</iconset>
   </property>
   <property name="text" >
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="helpAboutAction" >
   <property name="text" >
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc" />
 </resources>
 <connections/>
</ui>
